src/controllers/AuthController.js

```diff
@@ -183,3 +183,11 @@ export const resetPassword = catchAsync(async (req, res, next) => {
   // Send success response
   res.status(200).json({ message: 'Password has been successfully reset' });
 });
+
+export const currentUser = async (req, res) => {
```

Comment:
I think you might need to `import catchAsync from '../utils/catchAsync.js'` and use it before async() to catch any errors that occur. However, let's wait for today's meeting so we can hear Ellen's explanation first, which will help us gain a better understanding.